pkgs/development/python-modules/markdownsuperscript/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, markdown }:
+
+buildPythonPackage rec {
+  pname = "MarkdownSuperscript";
+  version = "2.0.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1dsx21h9hkx098d5azpw81dcz23rrgzrwlymwv7jri348q26p748";
+  };
+
+  propagatedBuildInputs = [ markdown ];
+
+  doCheck = false; # Tests depend on packages which are not available yet in nixpkgs
```

Comment:
Are you going to use and maintain this package? If not, then I'm of the opinion you should add test dependencies since otherwise there's no way to tell whether it is working.